boilerplate/app/components/ListItem.tsx

```diff
@@ -86,6 +86,11 @@ export interface ListItemProps extends TouchableOpacityProps {
    * Overrides `leftIcon`.
    */
   LeftComponent?: ReactElement
+
+  /**
+   * onPress handler for the ListItem
+   */
+  onPress?: () => void
```

Comment:
This isn't necessary as this type already extends `TouchableOpacityProps ` which includes `onPress`.